pyproject.toml

```diff
@@ -15,11 +15,15 @@ crewai = ">=0.28.8,<0.31.0"
 langchain-anthropic = "^0.1.11"
 langchain-groq = "^0.1.3"
 ollama = "^0.2.0"
+jupyter = "^1.0.0"
 
 [tool.poetry.group.dev.dependencies]
 pre-commit = "^3.7.0"
 ruff = "^0.4.3"
 
+[tool.ruff]
+exclude = ["llm_testing/test_inputs/test_functions_raw.py"]
+
```

Comment:
All settings for `ruff` should be in `.ruff.toml` - that way it's all in one place (you would put it here if that file didn't exist). See the `exclude` section.
It would also be a good idea to put _all_ test code files in one place, unless there's good reason to do otherwise.